### PR TITLE
Fix english doc api: sequence_slice, sigmoid_cross_entropy_with_logits, ssd_loss, DetectionMAP, detection_map

### PR DESCRIPTION
**Pull request by yghstill**
1. sequence_slice
2. sigmoid_cross_entropy_with_logits
3. ssd_loss
4. DetectionMAP
5. detection_map

image
image

image
image
image
image
image
image
image

### CODE REVIEW

File: python/paddle/fluid/layers/detection.py [hunk 1]
```diff
@@ -1400,32 +1404,30 @@ def ssd_loss(location,
             mining_type is 'hard_example'.
 
     Returns:
-        The weighted sum of the localization loss and confidence loss, with \
-        shape [N * Np, 1], N and Np are the same as they are in `location`.
+        Variable(Tensor), The weighted sum of the localization loss and confidence loss, \
```
Comment (wanghaoshuang) [inline python/paddle/fluid/layers/detection.py]: `Variable `写成`Variable :` 才能被正确解析出return type, 建议你预览一下格式

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -3228,12 +3228,12 @@ def sequence_slice(input, offset, length, name=None):
           should be equal. The **offset** should start from 0.
 
     Args:
-        input(Variable): The input Variable which consists of the complete
+        input(Variable): LoDTensor, The input Variable which consists of the complete
```
Comment (wanghaoshuang) [inline python/paddle/fluid/layers/nn.py:3223]: 请补充写支持的数据类型

File: python/paddle/fluid/layers/nn.py [hunk 3]
```diff
@@ -3228,12 +3228,12 @@ def sequence_slice(input, offset, length, name=None):
...
                          sequences.
-        offset(Variable): The offset to slice each sequence.
-        length(Variable): The length of each subsequence.
+        offset(Variable): LoDTensor, The offset to slice each sequence.
+        length(Variable): LoDTensor, The length of each subsequence.
         name(str|None): A name for this layer(optional). If set None, the
```
Comment (wanghaoshuang) [inline python/paddle/fluid/layers/nn.py]: `name`建议用统一的术语解释，如下：
The default value is None.  Normally there is no need for user to set this property.  For more information, please refer to :ref:`api_guide_Name`

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (yghstill) [reply python/paddle/fluid/layers/nn.py:3223]: DONE

File: python/paddle/fluid/layers/detection.py (hunk 1 above)
Comment (yghstill) [reply python/paddle/fluid/layers/detection.py]: DONE

File: python/paddle/fluid/layers/nn.py (hunk 3 above)
Comment (yghstill) [reply python/paddle/fluid/layers/nn.py]: DONE

**Review by wanghaoshuang — DISMISSED** [review]

**Review by wanghaoshuang — DISMISSED** [review]

**Review by wanghaoshuang — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Comment by heavengate** [comment]
注意以后commit message按照规范填写

**Pull request merged**